File: Python PowerPoint Examples/NotesSlidesApiDemo/deleteNotesSlide.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.notes_slides_api.NotesSlidesApi(configuration)

name = "DeleteNotesSlide.pptx"
password = ""
storage = ""
folder = "input"
slideIndex = 0

api.delete_notes_slide(name=name, slide_index=slideIndex, password=password, storage=storage,
                       folder=folder)

File: PHP PowerPoint Examples/NotesSlidesApiDemo/deleteNotesSlide.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\NotesSlidesApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new NotesSlidesApi($configuration);

$name = "DeleteNotesSlide.pptx";
$password = null;
$storage = null;
$folder = "NotesSlides";
$slideIndex = 0;
$apiInstance->deleteNotesSlide($name,$slideIndex, $password, $folder, $storage);
print_r($result);